### PR TITLE
Gems update

### PR DESCRIPTION
**Pull request by szymonbaranowski**

### CODE REVIEW

File: app/controllers/reminders_controller.rb [hunk 1]
```diff
@@ -67,7 +67,7 @@ def destroy
 
   def reminder_attrs
     params.require(:reminder)
-      .permit(:name, :valid_for_n_days, :remind_after_days,
-              :notification_text, :deadline_text, :slack_channel)
+          .permit(:name, :valid_for_n_days, :remind_after_days,
```
Comment (szymonbaranowski) [inline app/controllers/reminders_controller.rb]: how about no?

File: Gemfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 source "https://rubygems.org"
-gem "rails", "4.2.5"
+gem "rails", "4.2.5.1"
```
Comment (lubieniebieski) [inline Gemfile]: could you change this to +gem `~> 4` please?

Comment (szymonbaranowski) [inline Gemfile]: @lubieniebieski  fixed in 45060e52868577d8338f6f0da3418d93a6991bb3

**Comment by szymonbaranowski** [comment]
@lubieniebieski I added the rule to enforce comma after last parameter in literal, but also added #todo on [trello card](https://trello.com/c/mEDhuMfq/206-reminders-app-improvements) to fix a rubocop issue with keyword arguments

**Pull request merged**